src/config_builder.py

```diff
@@ -112,24 +118,28 @@ def _ruler(self) -> dict:
 
     @property
     def _schema_config(self) -> dict:
-        return {
-            "configs": [
-                {
-                    "from": "2020-10-24",
-                    "index": {"period": "24h", "prefix": "index_"},
-                    "object_store": "filesystem",
-                    "schema": "v11",
-                    "store": "boltdb-shipper",
-                },
+        configs = [
+            {
+                "from": "2020-10-24",
+                "index": {"period": "24h", "prefix": "index_"},
+                "object_store": "filesystem",
+                "schema": "v11",
+                "store": "boltdb-shipper",
+            }
+        ]
```

Comment:
Side question: would it be easier to always render tsdb, and have boltdb rendered conditionally?

Reply:
I would rather drop it off completely in the next breaking change, just to avoid on having these many conditions that depend on life-cycle events

Reply:
So `v12_migration_date` is not an optional arg.
Could you please take another look at this function? Perhaps there a better way to structure this?